src/containers/SubjectPage/SubjectPage.jsx

```diff
@@ -121,7 +140,7 @@ class SubjectPage extends Component {
                   </h1>
                   <TopicIntroductionList
                     toTopic={toTopic(subjectId)}
-                    topics={subjectTopics}
+                    topics={topics}
```

Comment:
Since subjectQuery uses `all` param. Every topic (including subtopics) is in the topic list.